pmsv2/virtualaccount/params.go

```diff
@@ -0,0 +1,54 @@
+package virtualaccount
+
+import (
+	"time"
+
+	"github.com/xendit/xendit-go/pmsv2/constant"
+)
+
+type ChannelCode string
+
+const (
+	BCA               ChannelCode = "BCA"
+	BJB               ChannelCode = "BJB"
+	BNI               ChannelCode = "BNI"
+	BRI               ChannelCode = "BRI"
+	MANDIRI           ChannelCode = "MANDIRI"
+	PERMATA           ChannelCode = "PERMATA"
+	BSI               ChannelCode = "BSI"
+	CIMB              ChannelCode = "CIMB"
+	SAHABAT_SAMPOERNA ChannelCode = "SAHABAT_SAMPOERNA"
+	ARTAJASA          ChannelCode = "ARTAJASA"
```

Comment:
can remove since integration is canceled for ARTAJASA

Reply:
removed